code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/KuptsovaAlesyaTest.java

```diff
@@ -0,0 +1,48 @@
+package ru.unn.agile.ComplexNumber.Model;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import ru.unn.agile.ComplexNumber.model.ComplexNumber;
+
+import static java.lang.String.valueOf;
+import static junit.framework.TestCase.assertEquals;
+
+public class KuptsovaAlesyaTest {
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    @Test
+    public void canNotCreateComplexNumberWithUncorrectValues() {
+        String re = "3.14";
+        String im = "-rjfho;i";
+        thrown.expect(IllegalArgumentException.class);
+        ComplexNumber z = new ComplexNumber(re, im);
+    }
+    @Test
+    public void canGetRealFromComplexNumberWithARealPart() {
+        String re = "3.14";
+        String im = "-13.0";
+        ComplexNumber z = new ComplexNumber(re, im);
+        double num = z.getReal();
+        String numstr = valueOf(num);
+        assertEquals(re, numstr);
+    }
+    @Test
+    public void canGetImaginaryFromComplexNumberWithImaginaryPart() {
+        String re = "3.14";
+        String im = "-13.0";
+        ComplexNumber z = new ComplexNumber(re, im);
+        double num = z.getImaginary();
+        String numstr = valueOf(num);
+        assertEquals(im, numstr);
+    }
+
+    @Test
+    public void canMultiplyNumbersWithNegativePartS() {
+        ComplexNumber z1 = new ComplexNumber(-1, 4);
+        ComplexNumber z2 = new ComplexNumber(-2, -3);
+        ComplexNumber num = z1.multiply(z2);
+        assertEquals(new ComplexNumber(14, -5), num);
```

Comment:
You need to take _new ComplexNumber(14, -5)_ to somethink like "expectedValue" variable,
and _num_ to "actualValue" variable everywhere